### PR TITLE
minor refactor

### PR DESCRIPTION
**Pull request by jpihl**
@janusheide merge if you agree :) There's not really anything worth writing in the news file IMO.

### CODE REVIEW

**Comment by janusheide** [comment]
I assume that the test_is_aligned.cpp file was removed by mistake? Otherwise looks good.

**Comment by jpihl** [comment]
I think it was originally added by mistake :) it's never compiled and I doubt it would since it's using the sak namespace.

**Comment by janusheide** [comment]
then it should be fixed to test the is_aligned function, currently I dont think there are any test of that function

**Comment by jpihl** [comment]
I was also a little confused about this file, but it seems it was left behind unintentionally.
The test is here:
https://github.com/steinwurf/allocate/blob/master/test/src/test_is_aligned.cpp
The left behind file is here:
https://github.com/steinwurf/allocate/blob/master/test/allocate/test_is_aligned.cpp

And since the test/allocate folder is not included in the test/wscript_build file we never saw the error.

**Comment by janusheide** [comment]
great, looks good

File: README.rst [hunk 1]
```diff
@@ -2,12 +2,25 @@
...
-.. contents:: Table of Contents:
-   :local:
+Our use-case for allocate is to ensure that if used with a std::vector then the
+buffer will start at the specified byte boundary.
+
+The allocator may be used with a std::vector like this::
+
+    std::vector<uint8_t, allocate::aligned_allocator<uint8_t>> data;
+
+The allocator defaults to 32 bit alignment - the lowest common denominotor on
```
Comment (petya2164) [inline README.rst]: should be: denominator

**Pull request merged**